react-native-libraries.json

```diff
@@ -7584,6 +7584,7 @@
   },
   {
     "githubUrl": "https://github.com/esthor/react-native-swipeable-list",
+    "npmpkg": "react-native-swipeable-list",
```

Comment:
```suggestion
```

This is redundant, since repository name matches npm package name:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg

Reply:
Whoops. Thanks. 🙏